data.sql

```diff
@@ -1,5 +1,11 @@
-/* Populate database with sample data. */
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Agumon', date '2020-02-03', 0, true, 10.23);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', date '2018-11-15', 2, true, 8.00);
```

Comment:
### Optional
- [ ] It would be nice if we do not include the “date” variable when we insert the data. Since we are writing the date as the values already

Reply:
Well noted. Thanks.